Lib/test/libregrtest/results.py

```diff
@@ -231,8 +231,7 @@ def display_summary(self, first_runtests: RunTests, filtered: bool):
             report.append(f'failures={stats.failures:,}')
         if stats.skipped:
             report.append(f'skipped={stats.skipped:,}')
-        report = ' '.join(report)
-        print(f"Total tests: {report}")
+        print(f"Total tests: {' '.join(report)}")
```

Comment:
There's the same issue here: `report` is a list earlier on in the function, but at this location, it's being assigned to a `str`. We could use a new variable name, or just do what I suggest here (make it an "anonymous variable")